src/renderer/pages/VideoEditorPage.vue

```diff
@@ -183,13 +206,43 @@ export default defineComponent({
                 startTime: 0.02,
                 volume: 255,
             },*/
-            ]
-        }
+            ],
+            thumbnailEvery:thumbnailFrequency
+        };
+        ipcRenderer.send('asynchronous-message', 
+        {
+          type: 'startEngine', 
+          data: engineOpts
+        });
+
+        const thumbEngineOpts = engineOpts;
+        thumbEngineOpts.outputType = 'thumbnail';
+        ipcRenderer.send('asynchronous-message', 
+        {
+          type: 'getThumbnails', 
+          data: thumbEngineOpts
         })
         });
     },
-    emits: ["open-single-editor", "open-recording-page", "recording"]
+    mounted() {
+        let timeline_h:any = (this.$refs.timeline as any).clientHeight;
+        let timeline_w: any = (this.$refs.timeline as any).clientWidth;
+
+        console.log(timeline_h)
```

Comment:
I understand the value of console.log for debugging, please remove to avoid unnecessary spam in the console for _other_ features